cpp/src/unary/math_ops.cu

```diff
@@ -433,6 +445,49 @@ struct MathOpDispatcher {
   }
 };
 
+template <typename UFN>
+struct BitwiseCountDispatcher {
+ private:
+  template <typename T>
+  static constexpr bool is_supported()
+  {
+    return std::is_integral_v<T>;
+  }
+
+  // Always use int32_t as output type for bit count.
+  using OutputType = int32_t;
+
+ public:
+  template <typename T, std::enable_if_t<is_supported<T>()>* = nullptr>
+  std::unique_ptr<cudf::column> operator()(cudf::column_view const& input,
```

Comment:
I believe column_views can be passed around by value
```suggestion
  std::unique_ptr<cudf::column> operator()(cudf::column_view input,
```

Reply:
I forgot the guideline for this, but I do see const ref used more often in the APIs.

Reply:
Yeah that is possible. But reference still smaller in size so we should prefer it over passing by value unless it is in device code (as I was told references work badly in device registers???).

Reply:
> I forgot the guideline for this, but I do see const ref used more often in the APIs.

I just looked into the codebase and noticed this too. Quite surprised as views should conceptually be lightweight enough to pass around by value. 😄. I think const ref is fine in that case then.